README.md

```diff
@@ -1,96 +1,179 @@
-by the CLI with your own.
-
-To run Remote Monitoring on your local machine:
-
-Once you become familiar with the solution you potentially will want to run 
-the solution locally.  You can do so through the following:
-1. Clone the repo with:
-   `git clone --recursive https://github.com/Azure/azure-iot-pcs-remote-monitoring-java`
-2. Install Docker: https://www.docker.com
-3. Create your IoT Hub either using the Azure Portal or by executing the IoT
-   Hub creation helper script in the repository:
-   [scripts/iothub/create-hub.sh](scripts/iothub/create-hub.sh)
-4. Create required environment variables: {TODO - need a script for this}
-5. Run the local deployment script: {TODO}
-
-Configuration
-=============
-
-Remote Monitoring uses environment variables for required configuration.
-Required environment variables are:
-- Name: PCS_IOTHUB_CONN_STRING
-  Value: <Your IoT Hub Connection String>
-- Name: PCS_IOTHUBREACT_CHECKPOINT_COSMOSDBSQL_CONNSTRING
-  Value: <Your DocumentDb Connection String>
-- {TODO - TCP ports for each msvc}
-
-
-Other documents
-===============
+Remote monitoring helps you get better visibility into your devices, assets, and
+sensors wherever they happen to be located. You can collect and analyze real-time
+device data using a remote monitoring solution that triggers automatic alerts and
+actions — from remote diagnostics to maintenance requests.
 
-* [Contributing](CONTRIBUTING.md)
-* [Development setup](DEVELOPMENT.md)
+[Azure IoT Hub](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-what-is-iot-hub)
+is a key building block of the remote monitoring sultion. IoT Hub is a fully
+managed service that enables reliable and secure bidirectional communications between
+millions of IoT devices and a solution back end.
 
+Check out the [Interactive Demo](http://dev-azureiotclicks01.azurewebsites.net/demos/remotemonitoring)
+for a detailed overview of features and use cases.
 
-[build-badge]: https://img.shields.io/travis/Azure/azure-iot-pcs-remote-monitoring-java.svg
-[build-url]: https://travis-ci.org/Azure/azure-iot-pcs-remote-monitoring-java
-[issues-badge]: https://img.shields.io/github/issues/azure/azure-iot-pcs-remote-monitoring-java.svg
-[issues-url]: https://github.com/azure/azure-iot-pcs-remote-monitoring-java/issues
+To get started you can folow along with the [Getting Started](#getting-started)
+for a command line deployment. You can also deploy using the web interface
+at https://azureiotsuite.com.
+
+>  This solution is currently in preview. The privew offers many features
+but there are known bugs. You can refer to our [Issues List](issues) in
+this repo (and in any submodules) to see known requests and issues. Feel
+free to submit new issues (and submit PRs for fixes if you'd like).
+
+Getting Started
+===============
+
+## Prerequisites
+* Azure Subscription ([permissions guidelines](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-permissions))
```

Comment:
when linking to microsoft.com try avoiding the "en-us" in the URLs, so that the user is redirected to a localized page when available. Please test the links though, because some microsoft.com sites don't offer the feature. In this case the link should be https://docs.microsoft.com/azure/iot-suite/iot-suite-permissions